code/game/jobs/job/central.dm

```diff
@@ -34,6 +34,9 @@
 		/obj/item/implant/dust
 	)
 	backpack = /obj/item/storage/backpack/satchel
+	backpack_contents = list(
+		/obj/item/grenade/bsa = 1
```

Comment:
Why do I feel like sooner or later, this is going to get thrown at an incompetent Captain.